code/datums/status_effects/debuffs/slime/slimed.dm

```diff
@@ -8,6 +8,38 @@
 	desc = "You are covered in slime and it's eating away at you! Find a way to wash it off!"
 	icon_state = "slimed"
 
+/atom/movable/screen/alert/status_effect/slimed/Click()
+	. = ..()
+	if (!.)
+		return FALSE
+	if (!can_wash())
+		return FALSE
+	INVOKE_ASYNC(src, PROC_REF(remove_slime))
```

Comment:
Update the desc to show this is clickable

Reply:
Or if you wanna do something fancy we can add an animation to clickable alerts, make it wiggle or something